LarkatorGUI/MainWindow.xaml

```diff
@@ -315,8 +316,8 @@
                             Visibility="{Binding CreateSearchAvailable, Converter={StaticResource BoolToVisibilityConverter}}"/>
                     <Grid Margin="16,0,16,16" Visibility="{Binding NewSearchActive, Converter={StaticResource BoolToVisibilityConverter}}" Background="#44ffffff">
                         <StackPanel Margin="8" Orientation="Vertical" HorizontalAlignment="Stretch" DataContext="{Binding NewSearch}">
-                            <ComboBox x:Name="groupsCombo" Text="{Binding Group}" IsEditable="True"/>
-                            <ComboBox x:Name="speciesCombo" Text="{Binding Species}"/>
+                            <ComboBox x:Name="groupsCombo" Text="{Binding Group}" IsEditable="True" />
+                            <ComboBox x:Name="speciesCombo" Text="{Binding Species}" IsEditable="True" />
```

Comment:
I am still trying to find a way to achieve this, I tried using the focus/enter events to trigger a function so I could programatically expand the list, but the event was not being triggered and the DroppedDown property is missing.  This is only my second time using C# so, I might be missing something to get that functionality working.

Reply:
Implemented in commit: 8efb4e016251de93a258653f2eb234d97a27e647

When the combobox gains keyboard focus the dropdown menu will expand.  If the combobox already has focus it will not expand.  To help this I implemented a mousedown event handler for the main window that will clear the current focus.  When clicking out of the combo box during my testing it took 2 clicks to actually clear the focus, but then clicking back in the combobox text area expanded the dropdown list again.